src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
       case f: WdlFile =>
-        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.toInt)
+        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.trim.toInt)
```

Comment:
Yeah, that's exactly it, I suppose if I read comments instead of code I'd have seen this sooner: `Unit method which closes the stream (if unset, close() will do nothing)